src/ai/backend/manager/models/gql.py

```diff
@@ -913,7 +913,9 @@ async def resolve_group(
                 ctx,
                 "Group.by_user",
             )
-            client_groups = await loader.load(client_user_id, type=[ProjectType[t] for t in type])
+            client_groups = [
+                group for group in await loader.load(client_user_id) if group.type in type
```

Comment:
On `groups` table definition, `type` column is non-nullable. May I ask why `null` check on `group.type` is necessary here?